File: services/categories.api/categories.api/Models/DTO/BlogpostModel.cs
namespace CodePulse.Api.Models.DTO
{
    public class BlogpostModel
    {
        public Guid Id { get;  set; }
        public string Title { get;  set; }
        public string UrlHandle { get;  set; }

        public string ShortDescription { get;  set; }
        public string Content { get;  set; }
        public string FeaturedImageUrl { get;  set; }
        public DateTime PublishDate { get;  set; }
        public string Auther { get;  set; }
        public bool IsVisible { get;  set; }
    }
}

File: web/categories/src/app/features/blog-post/models/blogpost.model.ts
import { Category } from "../../category/models/category.model";

export interface blogpost{
    id:string;
    title:string;
    urlhandle:string;
    shortDescription:string;
    content:string;
    featuredImageUrl:string; 
    publishDate:Date;
    auther:string;
    isVisible:boolean;
    categories:Category[];
}